examples/content.html

```diff
@@ -1,156 +1,412 @@
+			</p>
+		</details>
+		<details id="details-panel3">
+			<summary>Example 3</summary>
+			<p>
+				The story of the Norsemen runs thus. Towards the end of the ninth century, or nearly two hundred years before the Norman conquest, there was a great exodus or outswarming of the Norsemen from their original home in Norway. A certain King Harold had succeeded in making himself supreme in Norway, and great numbers of the lesser chiefs or jarls preferred to seek new homes across the seas rather than submit to his rule. So they embarked with their seafaring followers—Vikings, as we still call them—often, indeed, with their wives and families, in great open ships, and sailed away, some to the coast of England, others to France, and others even to the Mediterranean, where they took service under the Byzantine emperors. But still others, loving the cold rough seas of the north, struck westward across the North Sea and beyond the coasts of Scotland till they reached Iceland. This was in the year 874. Here they made a settlement that presently grew to a population of fifty thousand people, having flocks and herds, solid houses of stone, and a fine trade in fish and oil with the countries of Northern Europe. These settlers in Iceland attained to a high standard of civilization. They had many books, and were fond of tales and stories, as are all these northern peoples who spend long winter evenings round the fireside. Some of the sagas, or stories, which they told were true accounts of the voyages and adventures of their forefathers; others were fanciful stories, like our modern romances, created by the imagination; others, again, were a mixture of the two. Thus it is sometimes hard to distinguish fact and fancy in these early tales of the Norsemen. We have, however, means of testing the stories. Among the books written in Iceland there was one called the 'National Name-Book,' in which all the names of the people were written down, with an account of their forefathers and of any notable things which they had done.
+			</p>
+		</details>
+	</div>
+</div>
+<details class="mt-3">
+	<summary>Code</summary>
+	<pre><code>&lt;div class="wb-tabs vertical">
+	&lt;div class="tabpanels">
+		&lt;details id="details-panel1">
+			&lt;summary>Example 1&lt;/summary>
+			&lt;p>...&lt;/p>
+		&lt;/details>
+		&lt;details id="details-panel2" open="open">
+			&lt;summary>Example 2&lt;/summary>
+			&lt;p>...&lt;/p>
+		&lt;/details>
+		&lt;details id="details-panel3">
+			&lt;summary>Example 3&lt;/summary>
+			&lt;p>...&lt;/p>
+		&lt;/details>
+	&lt;/div>
+&lt;/div></code></pre>
+</details>
+
+<h2 id="cards">Cards</h2>
+<h3>Default card</h3>
+<div class="bg-light p-3">
+	<article class="oag-card">
+		<h3 class="mb-3"><a href="#">Lorem ipsum dolor sit amet</a></h3>
+		<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Veniam modi, ipsum sit adipisci expedita repellendus minus quidem maxime praesentium repellat.</p>
+	</article>
+</div>
```

Comment:
Where I can find a use case that math this example where the parent is a bg-light? Should the card only have a border by default when bg-light css are undefined? lets quickly talk about this one.